src/index.html

```diff
@@ -7,8 +7,62 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="icon">
+        <a href="#home" class="icon_link">
+          <img
+            src="images/logo.png"
+            alt="Moyo_icon"
+            class="icon"
+          >
+        </a>
+      </div>
+      <nav class="nav header_nav">
```

Comment:
```suggestion
      <nav class="nav header__nav">
```
check all such cases